cmd2/cmd2.py

```diff
@@ -5045,6 +5049,11 @@ def _current_script_dir(self) -> Optional[str]:
         help='record the output of the script as a transcript file',
         completer=path_complete,
     )
+    run_script_parser.add_argument(
+        '--history',
+        action=argparse.BooleanOptionalAction,
```

Comment:
`BooleanOptionalAction` was added in Python 3.9. Currently, cmd2 supports 3.7 and higher.
Consider changing this option name to `--no-history` and changing action to `store_true`.

Reply:
@laf0rge Would it be possible for you to make this change so it can work with Python 3.8?